services/s3control/src/it/java/software.amazon.awssdk.services.s3control/S3ControlIntegrationTest.java

```diff
@@ -0,0 +1,137 @@
+import software.amazon.awssdk.services.s3control.model.NoSuchPublicAccessBlockConfigurationException;
+import software.amazon.awssdk.services.s3control.model.PutPublicAccessBlockResponse;
+import software.amazon.awssdk.services.s3control.model.S3ControlException;
+import software.amazon.awssdk.services.sts.StsClient;
+import software.amazon.awssdk.testutils.service.AwsIntegrationTestBase;
+
+public class S3ControlIntegrationTest extends AwsIntegrationTestBase {
+
+    private String accountId;
+
+    private static final String INVALID_ACCOUNT_ID = "1";
+
+    private S3ControlClient client;
+
+    @Before
+    public void setup() {
+        StsClient sts = StsClient.create();
+        accountId = sts.getCallerIdentity().account();
+        client = S3ControlClient.builder()
+                                .overrideConfiguration(o -> o.addExecutionInterceptor(new AssertPayloadIsSignedExecutionInterceptor()))
+                                .build();
+    }
+
+    @After
+    public void tearDown() {
+        try {
+            client.deletePublicAccessBlock(DeletePublicAccessBlockRequest.builder().accountId(accountId).build());
+        } catch (Exception ignore) {
+            
+        }
+    }
+
+    @Test
+    public void putGetAndDeletePublicAccessBlock_ValidAccount() throws InterruptedException {
+        PutPublicAccessBlockResponse result =
+            client.putPublicAccessBlock(r -> r.accountId(accountId)
+                                              .publicAccessBlockConfiguration(r2 -> r2.blockPublicAcls(true)
+                                                                                      .ignorePublicAcls(true)));
+        assertNotNull(result);
+
+        // Wait a bit for the put to take affect
+        Thread.sleep(5000);
+
+        GetPublicAccessBlockResponse config = client.getPublicAccessBlock(r -> r.accountId(accountId));
+        assertTrue(config.publicAccessBlockConfiguration().blockPublicAcls());
+        assertTrue(config.publicAccessBlockConfiguration().ignorePublicAcls());
+
+        assertNotNull(client.deletePublicAccessBlock(r -> r.accountId(accountId)));
+    }
+
+    @Test
+    public void putPublicAccessBlock_NoSuchAccount() {
+        try {
+            assertNotNull(client.putPublicAccessBlock(r -> r.accountId(INVALID_ACCOUNT_ID)
+                                                            .publicAccessBlockConfiguration(r2 -> r2.restrictPublicBuckets(true))));
+            fail("Expected exception");
+        } catch (S3ControlException e) {
+            assertEquals("AccessDenied", e.awsErrorDetails().errorCode());
+            assertNotNull(e.requestId());
+        }
+    }
+
+    @Test
+    public void getPublicAccessBlock_NoSuchAccount() {
+        try {
+            client.getPublicAccessBlock(r -> r.accountId(INVALID_ACCOUNT_ID));
+            fail("Expected exception");
+        } catch (S3ControlException e) {
```

Comment:
Can we use ExpectedException instead of all this boilerplate try/catch/fail stuff.